maintenance_stock/tests/test_maintenance_stock.py

```diff
@@ -0,0 +1,148 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+import odoo.tests.common as test_common
+from odoo.tests.common import Form
+
+
+class TestMaintenanceStock(test_common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.maintenance_warehouse = self.env["stock.warehouse"].create(
+            {
+                "name": "Test warehouse",
+                "code": "TEST",
+            }
+        )
+
+        self.product1 = self.env["product.product"].create(
+            {
+                "default_code": "TESTOPROD",
+                "name": "Test prod",
+                "type": "product",
+                "uom_id": self.env.ref("uom.product_uom_unit").id,
+                "uom_po_id": self.env.ref("uom.product_uom_unit").id,
+            }
+        )
+
+        self.equipment_1 = self.env["maintenance.equipment"].create(
+            {
+                "name": "Test equipment",
+                "allow_consumptions": True,
+                "equipment_assign_to": "employee",
```

Comment:
```suggestion
```
@Reyes4711-S73 @dalonsod @etobella 
I think this field `equipment_assign_to ` should be removed to avoid the dependence of `hr_maintenance`.

Everything else looks very good.